content/graphql/guards-interceptors.md

```diff
@@ -96,11 +96,11 @@ GraphQLModule.forRoot({
 }),
 ```
 
-> **Warning** Enabling interceptors for field resolvers can cause performance issues when you are returning lots of records and your field resolver is executed thousands of times. For this reason, when you enable `fieldResolverEnhancers`, we advise you to skip execution of interceptors that are not strictly necessary for your field resolvers. You can do this using the following helper method:
+> **Warning** Enabling interceptors for field resolvers can cause performance issues when you are returning lots of records and your field resolver is executed thousands of times. For this reason, when you enable `fieldResolverEnhancers`, we advise you to skip execution of interceptors that are not strictly necessary for your field resolvers. You can do this using the following helper function:
 
 ```typescript
 export function isResolvingGraphQLField(context: ExecutionContext): boolean {
-  if (context.getType().toString() === 'graphql') {
+  if (context.getType<GqlContextType>().toString() === 'graphql') {
```

Comment:
I think you can simplify further (see http://localhost:4200/fundamentals/execution-context#current-application-context)

```suggestion
  if (context.getType<GqlContextType>() === 'graphql') {
```

Reply:
Had missed that from @kamilmysliwiec's comment.